src/pages/index.tsx

```diff
@@ -14,79 +14,91 @@ import Head from "next/head"
 import { ParsedUrlQuery } from "querystring"
 import listCourtCases from "useCases/listCourtCases"
 
+import type CourtCase from "entities/CourtCase"
+
 interface Props {
   user: User
   courtCases: CourtCase[]
   order: QueryOrder
   resultFilter?: Filter
   defendantNameFilter?: string
+  totalPages: number
+  pageNum: number
 }
 
+const validateQueryParams = (param: string | string[] | undefined): param is string => typeof param === "string"
+
 export const getServerSideProps = withMultipleServerSideProps(
   withAuthentication,
   async (context: GetServerSidePropsContext<ParsedUrlQuery>): Promise<GetServerSidePropsResult<Props>> => {
     const { currentUser, query } = context as AuthenticationServerSidePropsContext
-    const {
-      orderBy,
-      order,
-      resultFilter: resultFilterParam,
-      defendant
-    } = query as {
-      orderBy: string
-      order: string
-      resultFilter: string
-      defendant: string
-    }
-    const resultFilter = queryParamToFilterState(resultFilterParam)
+    const { orderBy, pageNum, resultFilter: resultFilterParam, defendant, maxPageItems, order } = query
+    const resultFilter = queryParamToFilterState(resultFilterParam as string)
+
+    const validatedMaxPageItems = validateQueryParams(maxPageItems) ? maxPageItems : "5"
+    const validatedPageNum = validateQueryParams(pageNum) ? pageNum : "1"
+    const validatedOrderBy = validateQueryParams(orderBy) ? orderBy : "ptiurn"
+    const validatedDefendantName = validateQueryParams(defendant) ? defendant : undefined
+    const validatedOrder: QueryOrder = order === "asc" ? "desc" : "asc"
 
     const dataSource = await getDataSource()
     const courtCases = await listCourtCases(dataSource, {
       forces: currentUser.visibleForces,
-      limit: 100,
-      orderBy: orderBy,
-      order: order as QueryOrder,
-      defendantName: defendant,
-      resultFilter: resultFilter
+      ...(validatedDefendantName && { defendantName: validatedDefendantName }),
+      resultFilter: resultFilter,
+      maxPageItems: validatedMaxPageItems,
+      pageNum: validatedPageNum,
+      orderBy: validatedOrderBy,
+      order: validatedOrder
     })
 
     if (isError(courtCases)) {
       throw courtCases
     }
 
-    const oppositeOrder: QueryOrder = order === "asc" ? "desc" : "asc"
```

Comment:
Moving this before line 45, where we call `listCourtCases` causes a small bug, while the URL param shows asc/desc the return result is the opposite:
![image](https://user-images.githubusercontent.com/22743709/181204630-4bdd1f6c-8fba-42a5-874c-bdce01c5444a.png)